src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/DataAccessSpec.scala

```diff
@@ -10,14 +10,19 @@ class DataAccessSpec extends TestDriverComponentWithFlatSpecAndMatchers {
   import driver.api._
 
   "DataAccess" should "test that truncateAll has left the DB in a known empty state" in withEmptyTestDatabase {
-    runAndWait(MTable.getTables).map(_.name.name) foreach { tableName =>
-      val safeTableName = tableName match {
-        case "GROUP" => "`GROUP`"
-        case "USER" => "`USER`"
-        case _ => tableName
-      }
+    val rawTableNames: Seq[String] = runAndWait(MTable.getTables).map(_.name.name)
+
+    val safeTableNames: Seq[String] = rawTableNames flatMap {
+      case "GROUP" => Option("`GROUP`")
+      case "USER" => Option("`USER`")
+      case "EXPREVAL_TEMP" => None            // temporary table
```

Comment:
I was seeing problems without this.  But that may have been before I removed it from `allSchemas`.  I'll run some tests.